app/services/admin/collections_csv_generator.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Admin
+  # Converts a set of collections to CSV.
+  class CollectionsCsvGenerator
+    def self.generate(collections)
+      new(collections).generate
+    end
+
+    def initialize(collections)
+      @collections = collections
+    end
+
+    def generate
+      CSV.generate do |csv|
+        csv << headers
+        collections.each do |collection|
+          csv << row(collection)
+        end
+      end
+    end
+
+    private
+
+    attr_reader :collections
+
+    def headers
+      [
+        'collection title',
+        'collection id',
+        'collection druid',
+        'state',
+        'version number',
+        'creator',
+        'managers',
+        'date created',
+        'date last modified',
+        'release setting',
+        'visibility setting',
+        'license',
+        'DOI yes/no',
+        'review workflow'
+      ]
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    def row(collection)
+      [
+        collection.head.name,
+        collection.id,
+        collection.druid,
+        collection.head.state,
+        collection.head.version,
+        collection.creator.sunetid,
+        collection.managed_by.map(&:sunetid).join(', '),
```

Comment:
Comma separation in a CSV string might be confusing. I might use a semicolon instead.